util/network-devp2p/src/host.rs

```diff
@@ -707,12 +710,17 @@ impl Host {
 								(info.config.min_peers as usize, max_peers as usize, info.config.non_reserved_mode == NonReservedPeerMode::Deny, info.id().clone())
 							};
 
+							max_peers = max(max_peers, min_peers);
```

Comment:
Why is this needed? (assuming we validate in the config that `max_peers >= min_peers`)

Reply:
Just another sanity check, so that this code does not rely on config.